src/sendgrid.lisp

```diff
@@ -14,6 +15,22 @@
 
 (defvar *verbose* nil)
 
+(defparameter *one-day-in-seconds* 85400 "one day less 1000 seconds")
```

Comment:
why less 1000 seconds? Just curious.

Reply:
Good question. Because if it were 4 full days (in epoch time), the sendgrid api will reject 4 days into the future. As it will be seen as the fifth day. This allows the user to send on the 4th day.